### PR TITLE
License counts are considered in the search field values API

### PR DESCRIPTION
**Pull request by RishiDiwanTT**
## Description
Open access and self hosted licenses are added unconditionally.
Otherwise the license pool must have more than owned licenses.

## Motivation and Context
The search API only displays items that have owned licenses of the work.
Due to this the count of works in the search field would be different than the actual results returned.
More details in [notion](https://www.notion.so/lyrasis/Search-Field-Values-API-does-not-account-for-unavailable-license-counts-9eab8a7e46ea4dbd90b14e6aff9cb287).

## Checklist

- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.

### CODE REVIEW

File: api/admin/controller/__init__.py [hunk 1]
```diff
@@ -495,7 +496,8 @@ def __call__(self, collection, book, path=None):
 
 class TimestampsController(AdminCirculationManagerController):
     """Returns a dict: each key is a type of service (script, monitor, or coverage provider);
-    each value is a nested dict in which timestamps are organized by service name and then by collection ID."""
+    each value is a nested dict in which timestamps are organized by service name and then by collection ID.
```
Comment (pcustic) [inline api/admin/controller/__init__.py:499]: Improving documentation one `.` at a time :D

File: tests/api/admin/controller/test_admin_search_controller.py [hunk 2]
```diff
@@ -101,3 +102,53 @@ def test_search_field_values(self, admin_search_fixture: AdminSearchFixture):
         assert response["languages"] == {"English": 2, "Spanish": 1, "Mandingo": 10}
         assert response["publishers"] == {"Publisher 1": 3, "Publisher 10": 10}
         assert response["distributors"] == {"Gutenberg": 13}
+
+    def test_no_license_counts(self, admin_search_fixture: AdminSearchFixture):
```
Comment (pcustic) [inline tests/api/admin/controller/test_admin_search_controller.py:106]: Nit: a bit confusing method name.

**Review by pcustic — APPROVED** [review]
Seems OK to me, one nitpick.

File: tests/api/admin/controller/test_admin_search_controller.py (hunk 2 above)
Comment (RishiDiwanTT) [reply tests/api/admin/controller/test_admin_search_controller.py]: :+1: changed this to `test_different_license_types`

**Pull request merged**